cli/pcluster/utils.py

```diff
@@ -359,9 +359,14 @@ def _get_instance_families_from_types(instance_types):
     return list(families)
 
 
-def _instance_types_and_families_are_supported(candidate_types_and_families, known_types_and_families):
-    """Return a boolean describing whether the given instance types and families are known to be supported."""
-    unknowns = [candidate for candidate in candidate_types_and_families if candidate not in known_types_and_families]
+def _batch_instance_types_and_families_are_supported(candidate_types_and_families, known_types_and_families):
+    """Return a boolean describing whether the instance types and families parsed from Batch API are known."""
+    known_exceptions = ["optimal"]
```

Comment:
optimal can't be specified in a comma separated list but it needs to be the only value provided right?

Reply:
that wasn't a constraint i was aware of. is that enforced by the Batch API?

i'll verify whether or not that's the case now.

Reply:
I was able to successfully create a cluster using `compute_instance_type = optimal,m3.xlarge`, so at least the compute environment can be created using those multiple values. I seem to be able to submit jobs as well.